romana-install/roles/stack/kubernetes/install-minion/templates/cni/romana-k8s-network

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Could we call this template "romana-cni-config" or "config" (since it's under a cni directory already)?

'romana-k8s-network' doesn't seem to reflect what this file is.